gnu.elf/plugin.xml

```diff
@@ -1450,6 +1552,14 @@
                   name="%linker.option.nano"
                   valueType="boolean">
             </option>
+            <option
```

Comment:
This should be an enumerated value in the drop down box, not a checkbox. `--specs=nano.specs` is a special case, because it can be combined with other specs files that provide board, however hl.specs provides the board, so it can't be combined with other boards. You need to create a new dropdown box with two boards, as mentioned in the original Jira:  --specs=hl.specs and --specs=qemu.specs.